ui-lib/src/main/java/co/electriccoin/zcash/ui/common/model/VersionInfo.kt

```diff
@@ -53,3 +58,8 @@ data class VersionInfo(
         }
     }
 }
+
+enum class DistributionDimension(val value: String) {
```

Comment:
Please add a comment to this and the other class about the fact that both need to be changed. If only one of them is, the app won't be installable.